BadApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BadApi.Controllers.Base;
+using BadApi.Domain;
+using BadApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+
+using Dapper;
+using BadApi.Models.Account;
+
+namespace BadApi.Controllers
+{
+	[Authorize]
+	[RoutePrefix("api/account")]
+    public class AccountController : BaseApiController
+    {
+        // GET: Account
+		[AllowAnonymous]
+		[HttpGet]
+		[Route("login")]
+        public async Task<Account> Login([FromUri] LoginModel model)
+        {
+            var connection = GetConnection();
+			var users = await connection.QueryAsync<Account>("SELECT * FROM dbo.Account;");
```

Comment:
A couple of things wrong here:
* fetching all columns using wildcard
* fetching all users to filter in memory